src/aiodynamo/fast/http/httpx.py

```diff
@@ -0,0 +1,30 @@
+import json
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+from httpx import AsyncClient
+from yarl import URL
+
+from ..errors import exception_from_response
+from .base import HTTP, Headers, RequestFailed
+
+
+@dataclass(frozen=True)
+class HTTPX(HTTP):
+    client: AsyncClient
```

Comment:
I'm guessing it would create an `AsyncClient()` if none is passed right?
It would be nice to add a small example of advanced use, e.g. how to pass connection pool, etc, parameters.

Reply:
You have to pass your own. The reason for this is that clients needs to be closed (usually), so that's the callers responsibility (not aiodynamo).

Reply:
example how to use it: https://github.com/hennge/aiodynamo/blob/e5fcb911a75431aae51e3b5080ba738b2f7a1646/tests/integration/test_client.py#L85-L87